tools-libselinux/patches/0001-libselinux-exclude-requirement-for-etc-selinux.patch

```diff
@@ -0,0 +1,46 @@
+From d759ab08ab7cfa88afb5fcac749b08e8e26aeab3 Mon Sep 17 00:00:00 2001
+From: Dmitry Sharshakov <dmitry.sharshakov@siderolabs.com>
+Date: Fri, 4 Oct 2024 21:08:24 +0200
+Subject: [PATCH] libselinux: exclude requirement for /etc/selinux
+
+We don't use this directory in Talos but load a policy embedded in the init binary
+
+Signed-off-by: Dmitry Sharshakov <dmitry.sharshakov@siderolabs.com>
+---
+ src/enabled.c | 4 ----
+ src/init.c    | 3 ---
+ 2 files changed, 7 deletions(-)
+
+diff --git a/src/enabled.c b/src/enabled.c
+index fefb0bd9..3e56f981 100644
+--- a/src/enabled.c
++++ b/src/enabled.c
+@@ -13,11 +13,7 @@ int is_selinux_enabled(void)
+ 	/* init_selinuxmnt() gets called before this function. We
```

Comment:
do we still need this patch?

Reply:
Yes, we do not create `/etc/selinux/config`